build.sbt

```diff
@@ -198,6 +198,21 @@ lazy val zincCompile = (project in file("zinc-compile"))
   )
   .configure(addSbtUtilTracking)
 
+def zincPersistMimaExcludes = {
```

Comment:
I am not sure if this is the best place to keep excludes so I am open for suggestions.

Reply:
Can you make it a mima exclude file as they have in the compiler for Scala reflect and the library?